database-commons/src/main/java/io/cdap/plugin/db/sink/AbstractDBSink.java

```diff
@@ -467,12 +467,14 @@ public abstract static class DBSinkConfig extends DBConfig implements DatabaseSi
     @Description("Name of the database schema of table.")
     @Macro
     @Nullable
-    private String dbSchemaName;
+    public String dbSchemaName;
```

Comment:
why did we change the scope of this variable?

Reply:
The variable `dbSchemaName` is directly accessed at two places outside the package.

1. `AuroraPostgresSinkConfig.getEscapedDbSchemaName()`

```
    public String getEscapedDbSchemaName() {
      return ESCAPE_CHAR + dbSchemaName + ESCAPE_CHAR;
    } 
```

2. `SapHanaSinkConfig.getEscapedDbSchemaName()`

```
    public String getEscapedDbSchemaName() {
      return ESCAPE_CHAR + dbSchemaName + ESCAPE_CHAR;
    }
```

Reply:
okay..